.gitignore

```diff
@@ -9,6 +9,7 @@ result-*
 /doc/NEWS.txt
 /doc/manual.html
 /doc/manual.pdf
+/nixpkgs.iml
```

Comment:
What's this for?

Reply:
It's the JetBrains IDEA module declaration file, which is created automatically when opening the project.

Reply:
```suggestion

# JetBrains IDEA module declaration file
/nixpkgs.iml
```

Git's intent was for these to be in user gitignores, but that's not what's done in practice, fwiw.

Reply:
It disappeared.

Reply:
Moved to #146607.